sympy/core/power.py

```diff
@@ -1629,7 +1628,11 @@ def e2int(e):
         # either b0 is bounded but neither 1 nor 0 or e is infinite
         # b -> b0 + (b - b0) -> b0 * (1 + (b/b0 - 1))
         o2 = order*(b0**-e)
-        z = (b/b0 - 1).simplify()
+        from sympy import AccumBounds
+        if isinstance(b0, AccumBounds):  # "XXX This can be removed and simply "z = (b - b0)/b0" would be enough when the operations on AccumBounds have been fixed."
```

Comment:
The comment would be better placed on a line of its own, or maybe on two lines as it is fairly long. It would also be good to refer to the issue https://github.com/sympy/sympy/issues/18795.